sdk/appconfiguration/azure-appconfiguration/README.md

```diff
@@ -58,12 +58,14 @@ Alternatively, get the connection string from the Azure Portal.
 
 Once you have the value of the connection string, you can create the AzureAppConfigurationClient:
 
+<!-- SNIPPET:hello_world_sample.create_app_config_client -->
 ```python
-from azure.appconfiguration import AzureAppConfigurationClient
```

Comment:
wont this import be still needed ?

Reply:
This is from hello_world_sample.create_app_config_client. I will work with the SDK owners to see how we want to coordinate the readme snippet with the samples.

Reply:
Using the method from utils.py in code snippet looks odd for me. Can you change to get the value by `os.getenv()` and remove `get_connection_string()`?